mapping_project/lib/custom_app_bar.dart

```diff
@@ -1,39 +1,24 @@
 import 'package:flutter/material.dart';
-import 'settings_page.dart';
-import 'lessons_page.dart';
-import 'directionsSummaryPage.dart';
-import 'myhomepage.dart';
-import 'tannoy.dart';
 
 class CustomAppBar {
   static void _pushSettingsPage(BuildContext context) {
-    Navigator.of(context).pushReplacement(
-      MaterialPageRoute(builder: (BuildContext context) => MyApp()),
```

Comment:
Do we need this still - I assume we can get rid of the lines: 

`Navigator.of(context).pushReplacement(
      MaterialPageRoute(builder: (BuildContext context) => MyApp()),);`

As we have the pushReplacementNamed instead

Reply:
I think I've already done that - you might have commented on the old code?

Reply:
ahhh yes - GitHub didn't show me that this morning!